package.json

```diff
@@ -96,8 +96,6 @@
     "jest-config": "^27.5.1",
     "package-json-type": "^1.0.3",
     "parcel": "^2.4.1",
-    "react": "^18.2.0",
-    "react-dom": "^18.2.0",
```

Comment:
The `react*` devDeps should not be removed. They are devDeps and are used during development and testing.
Unless a more recent version of Node changed something, peerDeps don't get installed during development and so this would fail CI (if it ran; it [needs fixing](https://github.com/agilgur5/react-signature-canvas/issues/109#issuecomment-2561507935) per my last comment in the issue)

Reply:
See also [below comment](https://github.com/agilgur5/react-signature-canvas/pull/111#discussion_r1900280183) on the `package-lock.json` after I added its necessary changes